2018/readme.md

```diff
@@ -0,0 +1,128 @@
+# 2018 File Generation Features
+
+## The FileGenerator Class 
```

Comment:
I think this section should be titled "overview" or something similar as it discusses how the code works but isn't related to a specific code part. The text is good and is what should be present at the top of the repo.

Reply:
Sure! Changes has been made in the last commit.